### PR TITLE
add some description for context

### PR DESCRIPTION
**Pull request by SKTT1Ryze**
When reading the spec about the PLIC on riscv ISA, I had some trouble understanding what the 'context' is referred to.  
So I searched for its meaning in the internet, then found some descriptions about it from `linux `repo.  
There' s the link: https://github.com/torvalds/linux/blob/5bfc75d92efd494db37f5c4c173d3639d4772966/Documentation/devicetree/bindings/interrupt-controller/sifive%2Cplic-1.0.0.yaml  
I think adding some descriptions about the `context` will help us understand this document better, so create this pr.  
Hope for being merged.

### CODE REVIEW

**Review by changab — COMMENTED** [review]
Sorry for the late response, I missed this.

**Comment by SKTT1Ryze** [comment]
Thanks for your suggestion. I have changed the description. Is there anything else that needs to be changed?

File: riscv-plic.adoc [hunk 1]
```diff
@@ -176,16 +176,17 @@ Each global interrupt can be enabled by setting the corresponding bit in the
...
+PLIC has 15872 Interrupt Enable blocks for the contexts. 
+
+The `context` is referred 
 to the specific privilege mode in the specific Hart of specific RISC-V processor 
-instance. How PLIC organizes interrupts for the contexts (Hart and privilege mode) 
+instance. For example, in an 4-core system with 2-way SMT, you have 8 harts and probably
+at least two privilege modes per hart: machine mode and supervisor mode.
+(ref: https://github.com/torvalds/linux/blob/5bfc75d92efd494db37f5c4c173d3639d4772966/Documentation/devicetree/bindings/interrupt-controller/sifive%2Cplic-1.0.0.yaml)
+
+How PLIC organizes interrupts for the contexts (Hart and privilege mode)
```
Comment (changab) [inline riscv-plic.adoc:187]: This is my typo when copy and paste. should be
"However PLIC organizes ..."

**Comment by changab** [comment]
> Thanks for your suggestion. I have changed the description. Is there anything else that needs to be changed?

I am good with this change with the fix to the typo I made.
Thanks

File: riscv-plic.adoc (hunk 1 above)
Comment (SKTT1Ryze) [reply riscv-plic.adoc]: I had fixed the typo, but why it should be `However` instead of `How` ? The original one before this pr is `How`.

Comment (changab) [reply riscv-plic.adoc]: Oops... my bad. I thought it was a typo. Please help me to change it back to "How".

Comment (SKTT1Ryze) [reply riscv-plic.adoc:187]: Ok, I had fixed it.

**Review by changab — APPROVED** [review]

**Comment by changab** [comment]
Thanks for helping on this.

**Pull request merged**